scripts/reduce_CG1D_config_template.json

```diff
@@ -0,0 +1,116 @@
+{
+    "facility": "$facility",
+    "instrument": "$instrument",
+    "ipts": "$ipts",
```

Comment:
@gecage952  does `substitute_template()` also takes care of substituting templated values `$facility`, `$instrument`, and `$ipts`?

Reply:
Yeah, the update_dict gets the values from `extract_info_from_path`